sql-parser/src/main/java/io/crate/sql/tree/ResetStatement.java

```diff
@@ -23,21 +23,29 @@
 
 import com.google.common.base.MoreObjects;
 import com.google.common.base.Objects;
+import io.crate.common.collections.Lists2;
 
 import java.util.List;
+import java.util.function.Function;
 
-public class ResetStatement extends Statement {
```

Comment:
can you maybe also adjust the related raw classes in AstBuidler, see `visitSet`(`Assignment`), `visitSetGlobal`, etc